app/services/acuant/acuant_client.rb

```diff
@@ -50,5 +74,36 @@ def merge_facial_match_and_liveness_response(facial_match_response, liveness_res
         },
       )
     end
+
+    def check_results(post_response, instance_id)
+      if post_response.success?
+        fetch_doc_auth_results(instance_id)
+      else
+        failure(post_response.errors.first, post_response.to_h)
+      end
+    end
+
+    def fetch_doc_auth_results(instance_id)
+      results_response = get_results(instance_id: instance_id)
+      handle_document_verification_failure(results_response) unless results_response.success?
+
+      results_response
+    end
+
+    def handle_document_verification_failure(get_results_response)
+      mark_step_incomplete(:document_capture)
```

Comment:
It looks like `handle_document_verification_failure` incomplete is not defined on the Acuant client. 

Do we want the Acuant client to be aware of steps? I was thinking we could have it focus on processing images and returning a response and let the caller focus on managing steps.

Reply:
yup - that was a copy/paste mistake. It's handled in the step.